### PR TITLE
Change default logging to INFO

### LINKED ISSUE #77: Default log level should be INFO
The default log level set in `st2chatops.env` is debug

```
# Log level
export HUBOT_LOG_LEVEL=debug
```

This makes st2chatops log messages every 2 minutes, containing every alias loaded. This results in large st2chatops.log files. 

I think it would be better to have a default log level 'INFO'.

### PR DESCRIPTION
**Pull request by Mierdin**
Closes #77

### CODE REVIEW

**Review by LindsayHill — APPROVED** [review]

**Comment by arm4b** [comment]
Could this potentially break any ChatOps CI or similar in our e2e tests @humblearner where we grep for specific log messages?
If so, we might need to inject `debug` loglevel in internal environment after this gets merged.

**Review by humblearner — CHANGES_REQUESTED** [review]
This will not only break chatopsCI, it will also break the self check script: https://github.com/StackStorm/st2chatops/blob/master/scripts/self-check.sh#L187 and it will add a step for new users, changing the debug level, who may have problem setting up st2chatops for the first time. It would be better to add step in docs to decrease verbosity.

**Comment by Mierdin** [comment]
This change was actually requested by a user, that the messages are too verbose, and I'm inclined to agree - INFO is a common default log level. I'd prefer that we make this change, and follow @armab suggestion of injecting a DEBUG loglevel for our CI use case.

On a related note, the CircleCI checks have passed thus far, so if we're expecting CI to break, maybe something need adjusted.

**Comment by humblearner** [comment]
@Mierdin: I have seen the conversation in #chatops community channel. I agree with you that the default log level should be `info`, but here it has been `debug` for a while, for a reason. Moreover, this is **not** related to CircleCI checks for this repo. This will cause failure in ChatopsCI (robot tests). 

**Please make changes  at following locations as well:**

1. ChatOps CI in our [e2e_tests](https://github.com/StackStorm/st2tests/tree/master/robotfm_tests/chatopsCI)

2. Troubleshooting chatops for new users using this [guide](https://docs.stackstorm.com/troubleshooting/basic_chatops_troubleshooting.html) and [self-check-script](https://github.com/StackStorm/st2chatops/blob/master/scripts/self-check.sh)


However, I still don't agree with this change, because:
- On one hand, as an advanced user, I can go ahead and change the log level in `st2chatops.env` file. 
- On the other, as a new user,  I first change the log level to `debug` and then use the script. (Unless we make changes in the troubleshooting guide and the self-check script.)

Also, note, you will still see `INFO` logs every two minutes about the loaded commands:

```
Aug 23 19:31:24 st2demo004 hubot[8914]: [Wed Aug 23 2017 19:31:24 GMT+0000 (UTC)] INFO Connecting...
Aug 23 19:31:24 st2demo004 hubot[8914]: [Wed Aug 23 2017 19:31:24 GMT+0000 (UTC)] DEBUG Loading adapter slack
Aug 23 19:31:22 st2demo004 systemd[1]: Started StackStorm service st2chatops.
Aug 23 19:31:22 st2demo004 systemd[1]: Stopped StackStorm service st2chatops.
Aug 23 19:31:22 st2demo004 systemd[1]: Stopping StackStorm service st2chatops...
Aug 23 19:31:17 st2demo004 hubot[29402]: [Wed Aug 23 2017 19:31:17 GMT+0000 (UTC)] INFO 7 commands are loaded
Aug 23 19:31:17 st2demo004 hubot[29402]: [Wed Aug 23 2017 19:31:17 GMT+0000 (UTC)] DEBUG Added command: demo reset diskspace on {{hostname}} - Reset diskspace demo
Aug 23 19:31:17 st2demo004 hubot[29402]: [Wed Aug 23 2017 19:31:17 GMT+0000 (UTC)] DEBUG Added command: disk clean {{hostname}} {{directory=/var/log}} - Run diskspace
clean up from chatops
Aug 23 19:31:17 st2demo004 hubot[29402]: [Wed Aug 23 2017 19:31:17 GMT+0000 (UTC)] DEBUG Added command: disk clean {{directory}} on {{hostname}} - Run diskspace clean
up from chatops
Aug 23 19:31:17 st2demo004 hubot[29402]: [Wed Aug 23 2017 19:31:17 GMT+0000 (UTC)] DEBUG Added command: pack show {{ pack }} - Show information about the pack from Sta
ckStorm Exchange.
Aug 23 19:31:17 st2demo004 hubot[29402]: [Wed Aug 23 2017 19:31:17 GMT+0000 (UTC)] DEBUG Added command: pack show <pack> - Show information about the pack from StackSt
orm Exchange.
Aug 23 19:31:17 st2demo004 hubot[29402]: [Wed Aug 23 2017 19:31:17 GMT+0000 (UTC)] DEBUG Added command: pack search {{ query }} - Search for packs in StackStorm Exchan
ge and other directories.
Aug 23 19:31:17 st2demo004 hubot[29402]: [Wed Aug 23 2017 19:31:17 GMT+0000 (UTC)] DEBUG Added command: pack search <query> - Search for packs in StackStorm Exchange and other directories.
Aug 23 19:31:17 st2demo004 hubot[29402]: [Wed Aug 23 2017 19:31:17 GMT+0000 (UTC)] DEBUG Added command: pack install {{ packs }} - Install/upgrade StackStorm packs.
Aug 23 19:31:17 st2demo004 hubot[29402]: [Wed Aug 23 2017 19:31:17 GMT+0000 (UTC)] DEBUG Added command: pack install <gitUrl>[,<gitUrl>] - Install/upgrade StackStorm packs.
Aug 23 19:31:17 st2demo004 hubot[29402]: [Wed Aug 23 2017 19:31:17 GMT+0000 (UTC)] DEBUG Added command: pack install <pack>[,<pack>] - Install/upgrade StackStorm packs.
Aug 23 19:31:17 st2demo004 hubot[29402]: [Wed Aug 23 2017 19:31:17 GMT+0000 (UTC)] DEBUG Added command: pack get {{ pack }} - Get information about installed StackStorm pack.
Aug 23 19:31:17 st2demo004 hubot[29402]: [Wed Aug 23 2017 19:31:17 GMT+0000 (UTC)] DEBUG Added command: pack get <pack> - Get information about installed StackStorm pa:
```

**Comment by LindsayHill** [comment]
Q: how many other apps ship with DEBUG logs on by default?

And if we are determined to ship with DEBUG, then surely we should at least sort out this other pack issue of no log rotate script?

**Comment by Mierdin** [comment]
Closing for now, since this is a bit stale. Feel free to pick up where I left off on the branch.

**Comment by arm4b** [comment]
The change itself is good. Maybe @blag can pick it up in future and update the st2tests as part of the initiative to rectify st2chatops.

File: st2chatops.env [hunk 1]
```diff
@@ -10,7 +10,7 @@ export NODE_TLS_REJECT_UNAUTHORIZED=0
 export EXPRESS_PORT=8081
 
 # Log level
-export HUBOT_LOG_LEVEL=debug
+export HUBOT_LOG_LEVEL=info
```
Comment (cognifloyd) [inline st2chatops.env]:
```suggestion
export HUBOT_LOG_LEVEL=${HUBOT_LOG_LEVEL:-info}
```

**Review by cognifloyd — CHANGES_REQUESTED** [review]
In order for the ansible smoke-test to succeed, it needs to be able to override this environment var. I suggest setting this to info only as a default if the var is not already set.

https://github.com/StackStorm/ansible-st2/blob/7dbcc49bc0adc3b2502b6f4caa07769f25fbf0b4/roles/st2smoketests/tasks/st2chatops.yml#L7-L8

File: st2chatops.env (hunk 1 above)
Comment (arm4b) [reply st2chatops.env]: Good catch! :+1:

**Review by cognifloyd — APPROVED** [review]
Looks good!

**Comment by cognifloyd** [comment]
re @humblearner's objection
> * On the other, as a new user,  I first change the log level to `debug` and then use the script. (Unless we make changes in the troubleshooting guide and the self-check script.)

Maybe the self-check script (which I have not used or reviewed in detail) needs to modify the log level on the fly similar to how the ansible smoke-test sets the variable:
https://github.com/StackStorm/ansible-st2/blob/7dbcc49bc0adc3b2502b6f4caa07769f25fbf0b4/roles/st2smoketests/tasks/st2chatops.yml#L7-L8

**Pull request closed without merging**